src/workerd/api/global-scope.c++

```diff
@@ -146,7 +146,6 @@ kj::Promise<DeferredProxy<void>> ServiceWorkerGlobalScope::request(kj::HttpMetho
     Worker::Lock& lock,
     kj::Maybe<ExportedHandler&> exportedHandler,
     kj::Maybe<jsg::Ref<AbortSignal>> abortSignal) {
-  throwIfInvalidHeaderValue(headers);
```

Comment:
Where'd it go?

Reply:
Not needed. This was part of the additional checking that was added for debugging on the issue a couple months ago. It's not needed here now.